### DISCUSSION TITLE
What is the difference between minizip and zlib?

### DISCUSSION THREAD

**Discussion by 168519** [nmoinvaz/minizip#519] [Q&A] [answered]
https://github.com/madler/zlib

zlib have no commit after 2017, Can I use minizip to replace zlib?

**Comment by nmoinvaz** [comment, marked as answer]
Minizip uses zlib to do compression/decompression. There are other zlib distributions (zlib-ng, cloudflare, intel).

**Comment by 168519** [comment]
OK，Thanks.
